### PR TITLE
use paragraphs for top level items, not lists JLBP-12

### PR DESCRIPTION
**Pull request by elharo**
@garrettjonesgoogle

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
Your update doesn't address my concerns about the formatting...

**Comment by garrettjonesgoogle** [comment]
So are you suggesting that you do some more work on the narrative flow? I agree that it needs more work.

**Comment by elharo** [comment]
Yes, it needs more work. I still think this PR improves the doc and should be merged rather than waiting for additional content.

**Comment by garrettjonesgoogle** [comment]
If you really want to commit the wording improvements, I'm willing to approve it with the top-level bullets added back...

File: docs/JLBP-12.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
-  - Incubating (no support expected)
-  - Stable (SLOs in place to handle issues)
-  - Maintenance (only bugs will be addressed)
-  - Deprecated / Abandoned / Inactive (no work is expected)
-- Clearly state the API and behavioral stability clients can expect from the
-  library surface. Example statements:
+Document the lifecycle status and corresponding level of support
+of the library. Examples include:
+
+  - Incubating: no support expected
```
Comment (suztomo) [inline docs/JLBP-12.md]:
```suggestion
- Incubating: no support expected
```

I don't think you want this indent for this list.

File: docs/JLBP-12.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+  - Stable: SLOs in place to handle issues
+  - Maintenance: only bugs will be addressed
+  - Deprecated: critical bugs might be addressed for a limited period of time
+  - Archived: no work is expected
+
+Clearly state the API and behavioral stability clients can expect from the
+library. For example you might say:
+
   - This API is stable and suitable for production use, aside from packages,
```
Comment (suztomo) [inline docs/JLBP-12.md]: I don't think you want this indent for this list.

File: docs/JLBP-12.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
     classes, and methods marked internal or unstable.
   - This API is beta and will not experience an API or behavior breaking change
     unless a bug or design flaw is uncovered.
   - This API is in development. Anything may change at any time. Stable products
     should not depend on it.
-- Document a feedback mechanism. For example:
+
+Provide and document a feedback mechanism. For example:
+
   - File an issue at https://github.com/...
```
Comment (suztomo) [inline docs/JLBP-12.md]: I don't think you want this indent for this list.

File: docs/JLBP-12.md (hunk 1 above)
Comment (elharo) [reply docs/JLBP-12.md]: done

File: docs/JLBP-12.md (hunk 3 above)
Comment (elharo) [reply docs/JLBP-12.md]: done

**Pull request merged**